network.go

```diff
@@ -184,6 +184,8 @@ func networkLogger() *logrus.Entry {
 
 // Attach for virtual endpoint bridges the network pair and adds the
 // tap interface of the network pair to the hypervisor.
+// TODO: This should be split to multiple end point types with
+// therr own attach methods
```

Comment:
But this is one implementation of the `Endpoint` interface already, I don't get the comment here.

Reply:
I'll clean up the typo above, @sboeuf.  For content, I think the intention of Manohar on this comment is that for a "virtual endpoint", there can by multiple ways that you'd end up attaching them to the hypervisor itself (bridge, TC, macvtap), and that this should be explicitly handled here.

Reply:
Ok I understand. This will be for a follow up PR, no need to discuss further on this point !